python/ray/_private/job_manager/job_manager.py

```diff
@@ -67,6 +71,7 @@ class JobStatusStorageClient:
     Handles formatting of status storage key given job id.
     """
     JOB_STATUS_KEY = "_ray_internal_job_status_{job_id}"
+    JOB_CHILD_PROCESS_PID_KEY = "_ray_internal_job_child_process_pid_{job_id}"
```

Comment:
Hmm what is this used for? I'd really prefer to not do any process management outside of the supervisor actor.

Note that you can't really rely on PIDs in failure scenarios anyways because they can be recycled by the OS (you might kill a different process accidentally).

Can we just use the fate sharing reaper process that @tchordia shared for simplicity?

Reply:
+1, i think that's much simpler and much less room for error